deployment/docker/REQUIREMENTS.txt

```diff
@@ -55,4 +55,5 @@ pyshp==1.2.12
 
 # for django flatpages
 django-ckeditor==5.6.1
-djangorestframework-gis==0.13
\ No newline at end of file
+django-crispy-forms==1.7.2
```

Comment:
also put it in the deployment/production/REQUIREMENTS

Reply:
done @dimasciput